### PR TITLE
mp3info: add page

### PR DESCRIPTION
**Pull request by MDecker-MobileComputing**
- [ ] The page (if new), does not already exist in the repository.
- [ ] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [ ] The page has 8 or fewer examples.
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [ ] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**
0.8.5a

### CODE REVIEW

File: pages/common/mp3info.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# mp3info
+
+> Viewer/editor for ID3v1 (but not ID3v2) tags of MP3 files.
+> More information: <http://www.ibiblio.org/mp3info>.
+
+- Show all ID3v1 tags of a MP3 file:
+
+`mp3info {{path/to/mp3file}}`
```
Comment (navarroaxel) [inline pages/common/mp3info.md:8]:
```suggestion
`mp3info {{path/to/file.mp3}}`
```

File: pages/common/mp3info.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+- Edit ID3v1 tags interactively:
+
+`mp3info -i {{path/to/mp3file}}`
```
Comment (navarroaxel) [inline pages/common/mp3info.md:12]:
```suggestion
`mp3info -i {{path/to/file.mp3}}`
```

File: pages/common/mp3info.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- Set values for ID3v1 tags in a MP3 file:
+
+`mp3info -a "{{artist_name}}" -t "{{song_title}}" -l "{{album_title}}" -y {{year}} -c "{{comment_text}}" {{path/to/mp3file}}`
```
Comment (navarroaxel) [inline pages/common/mp3info.md:16]:
```suggestion
`mp3info -a "{{artist_name}}" -t "{{song_title}}" -l "{{album_title}}" -y {{year}} -c "{{comment_text}}" {{path/to/file.mp3}}`
```

File: pages/common/mp3info.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+- Set number of track on album:
+
+`mp3info -n {{track_number}} {{path/to/mp3file}}`
```
Comment (navarroaxel) [inline pages/common/mp3info.md:20]:
```suggestion
`mp3info -n {{track_number}} {{path/to/file.mp3}}`
```

File: pages/common/mp3info.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+- Print list of numbers for all music genres:
+
+`mp3info -G`
+
+- Set music genre for MP3 file:
+
+`mp3info -g {{genre_number}} {{path/to/mp3file}}`
```
Comment (navarroaxel) [inline pages/common/mp3info.md:28]:
```suggestion
`mp3info -g {{genre_number}} {{path/to/file.mp3}}`
```

File: pages/common/mp3info.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+- Set number of track on album:
+
+`mp3info -n {{track_number}} {{path/to/mp3file}}`
+
+- Print list of numbers for all music genres:
+
+`mp3info -G`
+
+- Set music genre for MP3 file:
```
Comment (navarroaxel) [inline pages/common/mp3info.md:26]:
```suggestion
- Set music genre for a specific MP3 file:
```

File: pages/common/mp3info.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+- Edit ID3v1 tags interactively:
+
+`mp3info -i {{path/to/mp3file}}`
+
+- Set values for ID3v1 tags in a MP3 file:
+
+`mp3info -a "{{artist_name}}" -t "{{song_title}}" -l "{{album_title}}" -y {{year}} -c "{{comment_text}}" {{path/to/mp3file}}`
+
+- Set number of track on album:
```
Comment (navarroaxel) [inline pages/common/mp3info.md:18]:
```suggestion
- Set number of track on album for a specific MP3 file:
```

File: pages/common/mp3info.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+- Show all ID3v1 tags of a MP3 file:
+
+`mp3info {{path/to/mp3file}}`
+
+- Edit ID3v1 tags interactively:
+
+`mp3info -i {{path/to/mp3file}}`
+
+- Set values for ID3v1 tags in a MP3 file:
```
Comment (navarroaxel) [inline pages/common/mp3info.md:14]:
```suggestion
- Set values for ID3v1 tags in a specific MP3 file:
```

File: pages/common/mp3info.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
+# mp3info
+
+> Viewer/editor for ID3v1 (but not ID3v2) tags of MP3 files.
+> More information: <http://www.ibiblio.org/mp3info>.
+
+- Show all ID3v1 tags of a MP3 file:
```
Comment (navarroaxel) [inline pages/common/mp3info.md:6]:
```suggestion
- Show all ID3v1 tags of a specific MP3 file:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Review by navarroaxel — APPROVED** [review]

File: pages/common/mp3info.md [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+
+- Edit ID3v1 tags interactively:
+
+`mp3info -i {{path/to/file.mp3}}`
+
+- Set values for ID3v1 tags in a specific MP3 file:
+
+`mp3info -a "{{artist_name}}" -t "{{song_title}}" -l "{{album_title}}" -y {{year}} -c "{{comment_text}}" {{path/to/file.mp3}}`
+
+- Set number of track on album for a specific MP3 file:
```
Comment (marchersimon) [inline pages/common/mp3info.md:18]:
```suggestion
- Set the number of the track in the album for a specific MP3 file:
```

File: pages/common/mp3info.md [hunk 11]
```diff
@@ -0,0 +1,28 @@
...
+
+`mp3info -n {{track_number}} {{path/to/file.mp3}}`
+
+- Print list of numbers for all music genres:
```
Comment (marchersimon) [inline pages/common/mp3info.md:22]:
```suggestion
- Print a list of valid genres and their numeric codes:
```

File: pages/common/mp3info.md [hunk 12]
```diff
@@ -0,0 +1,28 @@
...
+
+`mp3info -G`
+
+- Set music genre for a specific MP3 file:
```
Comment (marchersimon) [inline pages/common/mp3info.md:26]:
```suggestion
- Set the music genre for a specific MP3 file:
```

**Review by marchersimon — APPROVED** [review]
Thanks @MDecker-MobileComputing

**Pull request merged**